Apps/Admin/Client/Pages/SupportPage.razor

```diff
@@ -38,6 +38,10 @@
     </MudGrid>
 </MudForm>
 
+<HgBannerFeedback Severity="@Severity.Normal" IsEnabled="HasWarning" TResetAction="MessageVerificationActions.ResetStateAction">
+    @MessageVerificationState.Value.WarningMessage
+</HgBannerFeedback>
```

Comment:
Severity in mockup seems to be "Info" - should confirm expected appearance with @mk08sh.

Reply:
Yes the mock-up shows a rather normal looking bar, not a warning looking bar, for the message. I will check with Nino and Mandad on that.